src/error.rs

```diff
@@ -107,6 +107,22 @@ impl fmt::Debug for Error {
                         wlnfl!(f, "err-yk-no-service-macos")?;
                         let url = "https://apple.stackexchange.com/a/438198";
                         wlnfl!(f, "rec-yk-no-service-macos", url = url)?;
+                    } else if cfg!(target_os = "openbsd") {
+                        wlnfl!(f, "err-yk-no-service-pcscd")?;
+                        let pkg = "pkg_add pcsc-lite";
+                        let service_enable = "rcctl enable pcscd";
+                        let service_start = "rcctl start pcscd";
+                        wlnfl!(f, "rec-yk-no-service-pcscd-bsd", pkg = pkg,
+                            service_enable = service_enable,
+                            service_start = service_start)?;
```

Comment:
The second commit needs a `cargo fmt` to fix its formatting.

Reply:
Fixed!